versioned_docs/version-2.10.x/security-authorization.md

```diff
@@ -38,6 +38,12 @@ By default, the broker treats the connection between a proxy and the broker as a
 
 Pulsar uses *Proxy roles* to enable the authentication. Proxy roles are specified in the broker configuration file, [`conf/broker.conf`](reference-configuration.md#broker). If a client that is authenticated with a broker is one of its ```proxyRoles```, all requests from that client must also carry information about the role of the client that is authenticated with the proxy. This information is called the *original principal*. If the *original principal* is absent, the client is not able to access anything.
 
+:::note
+
+Starting from 2.10.4, if a Proxy is not correctly configured to use a role that is in the `proxyRoles`, the connection will get rejected.
```

Comment:
This requirement was only introduced in 3.0.0. It isn't present for any prior versions.

Reply:
Oh, I thought the changes were all applied to those patch version branches. I will revert them.